packages/react-components/src/checkbox/src/Checkbox.css

```diff
@@ -240,13 +261,24 @@
     mask-size: var(--scale-delta) var(--scale-delta);
 }
 
+/* SIZING | MEDIUM */
+.o-ui-checkbox-md input {
+    width: 28px;
+    height: 28px;
+}
+
 /* SIZING | LARGE */
 /* SIZING | LARGE | LABEL */
 .o-ui-checkbox-lg .o-ui-checkbox-box:before {
     width: var(--scale-foxtrot);
     height: var(--scale-foxtrot);
 }
 
+.o-ui-checkbox-lg input {
+    width: 32px;
+    height: 32px;
```

Comment:
Should use `--scale-golf`

Reply:
the logic is 2x the border width (2 sides to a box) + 2 times the offset